erizo/src/erizo/media/ExternalOutput.h

```diff
@@ -94,7 +94,14 @@ class ExternalOutput : public MediaSink, public RawDataReceiver, public Feedback
   // composed of one or more partitions.  However, we don't seem to be sent anything but partition 0
   // so the second scheme seems not applicable.  Too bad.
   vp8SearchState vp8SearchState_;
-  bool needToSendFir_;
+  bool needToSendFir_, hasAudio, audioTimebaseSet;
```

Comment:
please use `need_to_send_fir_`, `has_audio_`and, `audio_timebase_set` here. We're trying to follow Google's style guide from now on so double check new variables in this PR (we'll change the current code in future PRs)